core/src/trezor/lvglui/scrs/components/button.py

```
@@ -12,7 +12,9 @@
 
 
 class NormalButton(lv.btn):
-    def __init__(self, parent, text=_(i18n_keys.BUTTON__NEXT), enable=True) -> None:
+    def __init__(
+        self, parent, text=_(i18n_keys.BUTTON__NEXT), enable=True, pressed_style=None
```

Ruff (0.11.9)

Do not perform function call `_` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)